homeassistant/components/onvif/parsers.py

```diff
@@ -409,3 +449,26 @@ async def async_parse_jobstate(uid: str, msg) -> Event:
         )
     except (AttributeError, KeyError):
         return None
+
+
+@PARSERS.register("tns1:Monitoring/Backup/Last")
```

Comment:
Can we move this under the `tns1:Monitoring/OperatingTime/LastReset` topic to keep things better organized?

Reply:
Done